homeassistant/components/zwave_js/services.yaml

```diff
@@ -29,3 +29,37 @@ set_lock_usercode:
       example: 1234
       selector:
         text:
+
+set_config_parameter:
+  description: Set a configuration parameter to a Z-Wave device. You can provide either an entity_id or device_id.
+  target:
+    device:
+      integration: zwave_js
+  fields:
+    entity_id:
```

Comment:
I'm confused? Is this the target? If so, it is a duplicate of target...

Reply:
While testing we discovered that without it, the developer tools did not show entity_id or device_id as possible inputs

Reply:
The automation editor indeed works without it. So we guessed we need both and it worked for both cases.
We allow both a list of id's as a single string in the service code

Reply:
Oh right, you can target other entities.

I'll make a suggestion on how to solve that 👍

Reply:
thanks!

Reply:
Maybe I should also note that the automation editor hides the device_id and entity_id inputs if target is selected, at least that's what I saw when testing with latest frontend